common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerTokenResponse.java

```diff
@@ -75,7 +85,7 @@ protected BrokerTokenResponse(Parcel in) {
     @Override
     public void writeToParcel(Parcel dest, int flags) {
         if (dest != null) {
-            dest.writeLong(getExpiresIn() != null ? getExpiresIn() : 0);
+            dest.writeLong(getExpiresIn());
```

Comment:
I'm assuming that getExpiresIn has been refactored to no longer return null?

Reply:
Thanks shane ,, this is an unintentional change, reverting it back